twincache/src/main/openapi/twincache.yaml

```diff
@@ -0,0 +1,77 @@
+openapi: 3.0.3
+
+info:
+  title: Cosmo Tech Twin cache Manager API
+  description: Cosmo Tech Twin cache Manager API
+  version: 0.0.1-SNAPSHOT
+
+servers:
+- url: 'https://api.azure.cosmo-platform.com'
+- url: 'http://localhost:4010'
+
+security:
+- oAuth2AuthCode: [ ]
+
+tags:
+- name: twincache
+  description: Twin Cache Management
+
+paths:
+  /massimport:
+    post:
+      operationId: mass_import
+      tags:
+        - twincache
+      description: Launch a mass import job
```

Comment:
Needs to have a Body data structure here:

graphInstanceId
storageSource (need to groom if it is possible to have an agnostic source here without Azure adherence)